### PR TITLE
better method names (#243 Part 1)

### PR DESCRIPTION
**Pull request by chkl**
The first part of #243

### CODE REVIEW

**Comment by casid** [comment]
Thanks Christian for the quick PR!

My only nitpick would be, could we add tests for the method names?

Otherwise, it looks good to me! @edward3h, you know the model extension a lot better, I'd say you should have the final say before merge :-)

**Review by kelunik — APPROVED** [review]

**Comment by edward3h** [comment]
I agree it could do with some tests. Otherwise it looks good to me.

**Comment by casid** [comment]
Merged, thanks for the PR @chkl!

**Pull request merged**
